spec/features/instant_sessions_spec.rb

```diff
@@ -9,6 +9,33 @@
     allow(CfeConnection).to receive(:assess).and_return(api_response)
   end
 
+  context "without conditional reveals for assets enabled", :assets_without_conditional_reveals do
```

Comment:
thinking about how the instant sessions are working made me realise that we haven't considered what the result page should look like now that we have made these changes. It isn't on the ticket but I think it could be an oversight.

We currently display £0.00 or whatever the value is for these asset fields on the results page and I think really it should be saying they're not relevant or not applicable - needs design thought!

Reply:
Have implemented simple design with skip_unless and question displayed (as per housing benefit)